seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -88,7 +88,7 @@
         "</main>"
       ],
       "tests": [
-        "assert(code.match(/<h2\\s+style\\s*=\\s*(\\'|\")\\s*color\\s*:\\s*(?:rgb\\(\\s*255\\s*,\\s*0\\s*,\\s*0\\s*\\)|rgb\\(\\s*100%\\s*,\\s*0%\\s*,\\s*0%\\s*\\)|red|#ff0000|#f00|hsl\\(\\s*0\\s*,\\s*100%\\s*,\\s*50%\\s*\\))\\s*\\;?(\\'|\")>\\s*CatPhotoApp\\s*<\\/h2>/), 'message: Your <code>h2</code> element should be red.');",
+        "assert(code.match($(\"h2\").css(\"color\") === \"rgb(255, 0, 0)\", 'message: Your <code>h2</code> element should be red.');",
```

Comment:
This one is missing a `)` after the color

Reply:
@dhcodes I'm going to need some help with this one, I'm not seeing what's missing here

Reply:
@systimotic you're right. The test wasn't working so I assumed it was similar to the others but actually I believe the issue here is `code.match` which is looking for regex. I think this should be:
`assert($(\"h2\").css(\"color\") === \"rgb(255, 0, 0)\", 'message: Your <code>h2</code> element should be red.');"`

Reply:
@dhcodes Ah, no, that was totally on purpose. I wasn't sloppy enough yet. 
😉
So sorry! Should be fixed now. This time I actually tested properly 😅